packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.js

```diff
@@ -53,6 +53,7 @@ export const DefaultUsage = () => /*html*/ `
                 ino-message="${text('ino-message', sampleText)}"
                 ino-action-text="${text('ino-action-text', 'Anlegen')}"
                 ino-timeout="${number('ino-timeout', 5000)}"
+                ino-type="${select('ino-type', ['', 'error', 'warning'], '')}"
```

Comment:
Examples in the storybook for `warning` and `error` would be nice

Reply:
✅